tests/framework/builder.py

```diff
@@ -49,10 +49,9 @@ def ssh_key(self):
         """Return ssh key artifact linked to the root block device."""
         return self._ssh_key
 
-    @property
-    def vm(self):
-        """Return the Microvm object instance."""
-        return self._vm
+    def __del__(self):
+        """Teardown the VM."""
+        self._vm.kill()
```

Comment:
```suggestion
        self.vm.kill()
```

Reply:
done, it seems that this wasn't caught by a test because python seems to ignore exceptions that happen inside destructors